ddtrace/contrib/pytest/plugin.py

```diff
@@ -431,9 +493,50 @@ def _get_test_class_hierarchy(item):
 def pytest_collection_modifyitems(session, config, items):
     if _CIVisibility.test_skipping_enabled():
         skip = pytest.mark.skip(reason=SKIPPED_BY_ITR_REASON)
+
+        items_to_skip_by_module = {}
+        current_suite_has_unskippable_test = False
+
         for item in items:
+            suite_is_unskippable = _is_suite_unskippable(item)
+            test_is_unskippable = _is_test_unskippable(item)
+
+            if suite_is_unskippable:
+                log.debug("Module %s is marked as unskippable", item.module)
```

Comment:
Should it, perhaps, be `log.debug("Suite %s in module %s is marked as unskippable", (item.name, item.module))`?

Reply:
In `pytest`, we don't really have a difference between module and suite, so I think it'd be a little redundant, and in this case, `item.name` is a test.

Although, I guess probably more to the point, these debug logs don't appear at this stage of the process unless you bend over backwards and change the plugin's code to set up a logger before `pytest` does anyway, so for now, they're almost more like comments in the code than anything.

Reply:
Resolving since I removed `suite_is_unskippable`.